podcasts/End of Year/Views/StoriesView.swift

```diff
@@ -62,7 +62,6 @@ struct StoriesView: View {
                 .padding()
                 .background(Color.black)
 
-            storySwitcher
```

Comment:
When loading, the story switcher was added to the screen. If the user tapped anywhere, this would crash the app because the model tried to go to the next story. While there wasn't any.